python/grass/temporal/datetime_math.py

```diff
@@ -791,8 +791,8 @@ def check_datetime_string(time_string, use_dateutil=True):
         # relative time. dateutil will interpret a single number as a valid
         # time string, so we have to catch this case beforehand
         try:
-            value = int(time_string)
-            return _("Time string seems to specify relative time")
+            if int(time_string):
+                return _("Time string seems to specify relative time")
```

Comment:
I think this is enough:
```suggestion
            int(time_string)
            return _("Time string seems to specify relative time")
```